tests/Tus/ClientTest.php

```diff
@@ -1349,10 +1350,11 @@ public function it_throws_exception_when_unable_to_create_resource() : void
         $checksum     = hash_file('sha256', $filePath);
         $guzzleMock   = m::mock(Client::class);
         $responseMock = m::mock(Response::class);
+        $requestMock  = m::mock(Request::class);
 
         $responseMock
             ->shouldReceive('getStatusCode')
-            ->once()
+            ->twice()
```

Comment:
`getStatusCode` is not expected to be called twice

Reply:
yes, with the mock it only fires once. With my original changes the Guzzle exception accessed this function also, so it got fired twice.
Its fixed now